shared/people/index.native.js

```diff
@@ -7,48 +7,40 @@ import {globalStyles, styleSheetCreate} from '../styles'
 import {isIOS} from '../constants/platform'
 
 const People = (props: Props) => (
```

Comment:
Can you check your file formatter tool? When I load this and format it with my prettier/eslint settings I get these diffs:

```
diff --git a/shared/people/index.native.js b/shared/people/index.native.js
index b9dfca5484..657aad6319 100644
--- a/shared/people/index.native.js
+++ b/shared/people/index.native.js
@@ -22,14 +22,17 @@ const People = (props: Props) => (
           label: 'Avatar',
         },
       ]}
-      titleComponent={(<PeoplePageSearchBar {...props} />)}
+      titleComponent={<PeoplePageSearchBar {...props} />}
     />
     <Kb.ScrollView
       style={styles.scrollView}
       refreshControl={
         // TODO set refreshing to the actual prop once the bug in RN gets fixed
         // see https://github.com/facebook/react-native/issues/5839
-        <Kb.NativeRefreshControl refreshing={isIOS ? false : props.waiting} onRefresh={() => props.getData()} />
+        <Kb.NativeRefreshControl
+          refreshing={isIOS ? false : props.waiting}
+          onRefresh={() => props.getData()}
+        />
       }
     >
       <PeoplePageList {...props} />
```